### DISCUSSION TITLE
what is the password for IPOverWIFIBroadcase

### DISCUSSION THREAD

**Discussion by iamfaith** [svpcom/wfb-ng#180] [Questions]
![image](https://user-images.githubusercontent.com/16201784/143581810-4a603b80-f010-4322-b1a7-281751563511.png)

as the screenshot described.

**Comment by svpcom** [comment]
10.5.0.1 -- GS side
10.5.0.2 -- drone side

you can change password via standard `passwd` command

**Reply by iamfaith** [reply]
Please help me. Any replies will appreciate a lot. Thanks for your wonderful work.

**Comment by iamfaith** [comment]
![image](https://user-images.githubusercontent.com/16201784/143585985-ca38e984-6b4e-4f02-bdab-a0698a232531.png)

I am using the ground station to ping the drone side. It does not work.
![image](https://user-images.githubusercontent.com/16201784/143586140-d82a1662-e165-480e-8826-00301f3249c1.png)

**Reply by iamfaith** [reply]
Also, I can not ssh to the drone but only work for ssh to gs side when I am using gs side.

**Reply by svpcom** [reply]
Check that WFB link is up (video and telemetry streams works). If they works then check firewall settings on drone. Also check `sudo journalctl -xu wifibroadcast@wlanX` where wlanX is your WFB wifi card

**Reply by iamfaith** [reply]
I check that the video works. I am using python to run server.py.

**Reply by iamfaith** [reply]
> Check that WFB link is up (video and telemetry streams works). If they works then check firewall settings on drone. Also check `sudo journalctl -xu wifibroadcast@wlanX` where wlanX is your WFB wifi card

Is it because the destination is itself so that I can not ping the other side?
 inet 10.5.0.1  netmask 255.255.255.0  destination 10.5.0.1

**Reply by svpcom** [reply]
run server under root user

**Reply by iamfaith** [reply]
> run server under root user

All of them are under root user. The video is working perfectly.

**Reply by svpcom** [reply]
run `ifconfig -a` both on drone and gs

**Reply by svpcom** [reply]
Disable firewall via: `iptables -F; iptables -X`

**Reply by svpcom** [reply]
check that both sides have the same version of WFB

**Reply by iamfaith** [reply]
This is gs side:

root@kali:~/wifi_vtx# ifconfig -a
bond0: flags=5122<BROADCAST,MASTER,MULTICAST>  mtu 1500
        ether 1e:34:d2:42:38:1a  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

dummy0: flags=195<UP,BROADCAST,RUNNING,NOARP>  mtu 1500
        inet6 fe80::981c:66ff:fe35:7af0  prefixlen 64  scopeid 0x20<link>
        ether 9a:1c:66:35:7a:f0  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 840 (840.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

gs-wfb: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1446
        inet 10.5.0.1  netmask 255.255.255.0  destination 10.5.0.1
        inet6 fe80::af04:9ca6:ec1f:a1d7  prefixlen 64  scopeid 0x20<link>
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 500  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 11  bytes 640 (640.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

ip6_vti0: flags=128<NOARP>  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

ip6tnl0: flags=128<NOARP>  mtu 1452
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

ip_vti0: flags=128<NOARP>  mtu 1480
        tunnel   txqueuelen 1  (IPIP Tunnel)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1  (Local Loopback)
        RX packets 245321  bytes 270318140 (257.7 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 245321  bytes 270318140 (257.7 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

p2p0: flags=4098<BROADCAST,MULTICAST>  mtu 1500
        ether 02:0a:f5:b7:d2:ec  txqueuelen 3000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data0: flags=65<UP,RUNNING>  mtu 1500
        inet6 fe80::d3e5:bc65:d96e:bb0c  prefixlen 64  scopeid 0x20<link>
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 996 (996.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data1: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data2: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data3: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data4: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data5: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data6: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data7: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

r_rmnet_data8: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data0: flags=65<UP,RUNNING>  mtu 2000
        inet6 fe80::9663:6380:2964:c344  prefixlen 64  scopeid 0x20<link>
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 17  bytes 1322 (1.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 1275 (1.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data1: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data2: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data3: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data4: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data5: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data6: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data7: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data8: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data9: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_data10: flags=0<  mtu 1500
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

rmnet_ipa0: flags=65<UP,RUNNING>  mtu 2000
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 13  bytes 1526 (1.4 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 33  bytes 2543 (2.4 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

sit0: flags=128<NOARP>  mtu 1480
        sit  txqueuelen 1  (IPv6-in-IPv4)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.31.219  netmask 255.255.255.0  broadcast 192.168.31.255
        inet6 fe80::9cfb:6ff:fe7a:9291  prefixlen 64  scopeid 0x20<link>
        ether 9e:fb:06:7a:92:91  txqueuelen 3000  (Ethernet)
        RX packets 114889  bytes 140747604 (134.2 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 63384  bytes 8677776 (8.2 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        unspec 00-1C-50-0E-49-4E-67-73-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 273283  bytes 360101303 (343.4 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0



This is the drone side:
drone-wfb: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1446
        inet 10.5.0.2  netmask 255.255.255.0  destination 10.5.0.2
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 500  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        ether dc:a6:32:cf:16:a7  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 359836  bytes 404907334 (386.1 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 359836  bytes 404907334 (386.1 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

usb0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 192.168.3.199  netmask 255.255.255.0  broadcast 192.168.3.255
        ether 00:dd:dc:eb:6d:a1  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.31.69  netmask 255.255.255.0  broadcast 192.168.31.255
        ether dc:a6:32:cf:16:a9  txqueuelen 1000  (Ethernet)
        RX packets 2511  bytes 533453 (520.9 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 2484  bytes 879139 (858.5 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan1: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        unspec E8-4E-06-5E-54-31-00-00-00-00-00-00-00-00-00-00  txqueuelen 1000  (UNSPEC)
        RX packets 33  bytes 5370 (5.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 34  bytes 5236 (5.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

**Reply by iamfaith** [reply]
> iptables -F; iptables -X

All checked. And the firewall is shut down.

**Reply by svpcom** [reply]
Do you have any config customizations?

**Reply by svpcom** [reply]
Check that driver and card on GS support packet injection

**Reply by iamfaith** [reply]
> Check that driver and card on GS support packet injection

Could you please tell me which command should I use?

**Reply by svpcom** [reply]
modinfo 88XXau

**Reply by svpcom** [reply]
`ethtool -i wlanX`

**Reply by iamfaith** [reply]
> modinfo 88XXau

I just use `aireplay-ng --test` on both sides. They show injection is working.

**Reply by iamfaith** [reply]
> ethtool -i

pi@raspberrypi:~ $ ethtool -i wlan1
driver: mt76x2u
version: 5.10.17-v7l+
firmware-version: 0.0.00-b1
expansion-rom-version:
bus-info: 2-1:1.0
supports-statistics: yes
supports-test: no
supports-eeprom-access: no
supports-register-dump: no
supports-priv-flags: no
pi@raspberrypi:~ $ modinfo mt76x2u
filename:       /lib/modules/5.10.17-v7l+/kernel/drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2u.ko
license:        Dual BSD/GPL
author:         Lorenzo Bianconi <lorenzo.bianconi83@gmail.com>
firmware:       mt7662_rom_patch.bin
firmware:       mt7662.bin
srcversion:     034AC454D66FD385A67B135
alias:          usb:v045Ep02FEd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v045Ep02E6d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0846p9053d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v2C4Ep0103d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0E8Dp7632d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v7392pB711d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v057Cp8503d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0E8Dp7612d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p180Bd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p17EBd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p1833d*dc*dsc*dp*ic*isc*ip*in*
depends:        mt76x02-lib,mt76x2-common,mt76,mt76-usb,mac80211,mt76x02-usb
intree:         Y
name:           mt76x2u
vermagic:       5.10.17-v7l+ SMP mod_unload modversions ARMv7 p2v8

**Reply by iamfaith** [reply]
Here is gs side:

ethtool -i wlx1831bf54e99e
driver: mt76x2u
version: 5.4.0-53-generic
firmware-version: 0.0.00-b1
expansion-rom-version:
bus-info: 1-3:1.0
supports-statistics: yes
supports-test: no
supports-eeprom-access: no
supports-register-dump: no
supports-priv-flags: no


modinfo mt76x2u
filename:       /lib/modules/5.4.0-53-generic/kernel/drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2u.ko
license:        Dual BSD/GPL
author:         Lorenzo Bianconi <lorenzo.bianconi83@gmail.com>
firmware:       mt7662_rom_patch.bin
firmware:       mt7662.bin
srcversion:     E89066E16A0E2CA46727D80
alias:          usb:v045Ep02FEd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v045Ep02E6d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0846p9053d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v7392pB711d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v057Cp8503d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0E8Dp7612d*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p180Bd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p17EBd*dc*dsc*dp*ic*isc*ip*in*
alias:          usb:v0B05p1833d*dc*dsc*dp*ic*isc*ip*in*
depends:        mt76x02-lib,mt76x2-common,mt76,mt76-usb,mac80211,mt76x02-usb
retpoline:      Y
intree:         Y
name:           mt76x2u
vermagic:       5.4.0-53-generic SMP mod_unload
signat:         PKCS#7
signer:
sig_key:
sig_hashalgo:   md4

**Comment by svpcom** [comment]
What is `mt76x2u`? WFB supports only rtl8812au

**Reply by iamfaith** [reply]
Asus AC55. It supportss monitor mode and aircrack is working well. Why does WFB only support rtl8812au

**Reply by svpcom** [reply]
It may work but I only test WFB with 8812au. So you need to debug it yourself
